lib/cucumber/formatter/html_builder.rb

```diff
@@ -51,6 +41,18 @@ def format_features!(features)
 
       private
 
+      def string_to_embed(type: nil, src: nil, label: nil, id: nil)
+        raise ::ArgumentError, 'missing required argument' unless type && src && label && id # for Ruby 2.0 compatibility
+        raise InvalidEmbedTypeError, 'Invalid type. Valid types are :text and :image.' unless [:text, :image].include? type
```

Comment:
If you wish to golf more this error could introduce a VALID_EMBED_TYPES constant, pull up the interpolated message into its own constructor, and `super(that_interpolated_message)` there. 

This line could then `unless VALID_EMBED_TYPES.include?(type)`.

If this appeals to you, go for it, if not, let it go.

Reply:
Sure, I like that idea. Thanks!